packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md

```diff
@@ -2,7 +2,12 @@
 title: What is UniRep?
 ---
 
-# 👏 Welcome
+# Introduction
+
+
+UniRep is a private, non-repudiable data storage and retrieval system designed as a base layer for building custom and interoperable zk applications. It serves as a memory layer, allowing users to receive data, prove facts about their data, and store results of large proofs with strong privacy guarantees and constant computation complexity. 
```

Comment:
actually it stores results in a constant size of proof
not *large proofs*.

It could be `store results of the data with strong privacy guarantees...`

Reply:
I think "large proofs" refers to proving time/complexity